src/service/poc_mobile.proto

```diff
@@ -7,6 +7,81 @@ import "mapper.proto";
 import "hex_boosting.proto";
 import "service_provider.proto";
 
+message hex_usage_counts_req_v1 {
```

Comment:
I'm curious, do that data sources that drive hex usage include data transfer?  If so, would it be feasible to include the volume (bytes) of data transfer in a given hex as part of this payload? We currently aggregate data transfer leveraging hotspot locations - wondering if there could be a better way.

![image](https://github.com/user-attachments/assets/d2154507-a72c-49cf-8ced-cd29619f6a16)

Reply:
It is certainly possible to include the volume in bytes of data transfer on a per hex basis.  I understood you only wanted this per radio, I can include per hex also

Reply:
added here: https://github.com/helium/proto/pull/428/commits/238ac3e532215b35f719daba3b9fa59079318388